ocaml/lambda/translcore.ml

```diff
@@ -1095,7 +1095,7 @@ and transl_exp0 ~in_new_scope ~scopes sort e =
           (* Might be a module, which are all values.  Otherwise raise. *)
           ignore (Env.find_module_lazy path e.exp_env)
       ) arg_idents;
-      let body = Lambda.rename map lam in
+      let body = Lexclave (Lambda.rename map lam) in
```

Comment:
It's probably slightly easier to just remove the call to to `maybe_region_layout` below, since that will just remove the `Lexclave`. I'd probably set `region` to `false` rather than `true` as well, although I don't think it should matter.

Reply:
I talked to @mshinwell and turns out my understanding of `region` is wrong. It's outdated anyway and is removed in #2985 which I will review.